src/features/transfer-ownership/TransferOwnership.utils.ts

```diff
@@ -0,0 +1,15 @@
+import { isValidEthAddress } from '../../lib/util/address/address';
+
+export const isValid = (value: string, account: string) =>
+	isValidEthAddress(value) && !(value.toLowerCase() === account.toLowerCase());
+
+export const handleInputError = (
```

Comment:
Maybe could change the name of this to `getInputErrorMessage` to make it a bit more explicit, as it isn't handling the error per se.

Reply:
ah thats my bad, I moved it out of one file which lost its context. Makes sense to name ```getInputErrorMessage```